B827EBFFFE04D12D.json

```diff
@@ -0,0 +1,18 @@
+{
+    "gateway_conf" : {
+         "gateway_ID" : "B827EBFFFE04D12D",
+         "servers" : [
+              {
+                   "server_address" : "ttn-router-us-west",
```

Comment:
That's not a valid address, please check the list linked on the readme of this repo.